content/applications/finance/payment_providers/mercado_pago.rst

```diff
@@ -32,5 +41,14 @@ Configuration on Odoo
    at the step :ref:`payment_providers/mercado_pago/configure_dashboard`.
 #. Configure the rest of the options to your liking.
 
+.. image:: mercado_pago/mp-access-token.png
+   :align: center
+   :alt: Access token configuration for Mercado Pago in Odoo.
+
+.. important::
+   Mercado Pago functions correctly with the :guilabel:`Enabled` status activated for production and
+   testing purposes. For this payment provider the *Test Mode* configuration in Odoo will result in
+   an error message.
```

Comment:
I was not aware of this but it's not normal. Please create a support ticket so we can fix it instead :)